ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -162,16 +162,16 @@ func createCluster(context *cli.Context, rdwr config.ReadWriter, ecsClient ecscl
 
 	// Check if cluster is specified
 	if ecsParams.Cluster == "" {
-		return fmt.Errorf("Please configure a cluster using the configure command.")
+		return fmt.Errorf("Please configure a cluster using the configure command or with '--%s' flag", command.ClusterFlag)
```

Comment:
"Please configure a cluster using the configure command or the '--%s' flag"

Why are we getting rid of the periods here? Seems like these messages should have terminal punctuation. Especially the messages that have two or more sentences, since the first sentence has a period and the second doesn't.